### PR TITLE
TableAggregation: using stream of rows with specified typeinformation…

### PR DESCRIPTION
**Pull request by arkadius**
… instead of inferred typeinfo

## Checklist before merge
- [ ] Related issue ID is placed at the beginning of PR title in \[brackets\] (can be GH issue or Nu Jira issue)
- [ ] Code is cleaned from temporary changes and commented out lines
- [ ] Parts of the code that are not easy to understand are documented in the code
- [ ] Changes are covered by automated tests
- [ ] Showcase in dev-application.conf added to demonstrate the feature
- [ ] Documentation added or updated
- [ ] Added entry in _Changelog.md_ describing the change from the perspective of a public distribution user
- [ ] Added _MigrationGuide.md_ entry in the appropriate subcategory if introducing a breaking change
- [ ] Verify that PR will be squashed during merge

### CODE REVIEW

File: engine/flink/components/base-tests/src/test/scala/pl/touk/nussknacker/engine/flink/table/SpelValues.scala [hunk 1]
```diff
@@ -2,38 +2,39 @@ package pl.touk.nussknacker.engine.flink.table
...
+  val spelBoolean = "true"
+  val spelStr     = "'str'"
+  val spelChar    = "'1'.charAt(0)"
+
+  val spelByte    = "123.byteValue"
+  val spelShort   = "123.shortValue"
+  val spelInt     = "123"
+  val spelFloat   = "123.12.floatValue"
+  val spelDouble  = "123.12.doubleValue"
+  val spelBigInt  = "T(java.math.BigInteger).ONE"
```
Comment (raphaelsolarski) [inline engine/flink/components/base-tests/src/test/scala/pl/touk/nussknacker/engine/flink/table/SpelValues.scala:14]: imo despite the misleading name `long` java type was the valid type here for DataTypes.BIGINT column.

Comment (arkadius) [reply engine/flink/components/base-tests/src/test/scala/pl/touk/nussknacker/engine/flink/table/SpelValues.scala]: Yeah, you are right. I've been cheated by variable names. Now the name is as the java type, not a table API type.

**Review by raphaelsolarski — APPROVED** [review]

File: engine/flink/components/base-tests/src/test/scala/pl/touk/nussknacker/engine/flink/table/sink/TableFileSinkTest.scala [hunk 2]
```diff
@@ -65,6 +67,13 @@ class TableFileSinkTest extends AnyFunSuite with FlinkSpec with Matchers with Pa
       |      'format' = 'csv'
       |) LIKE input;
       |
+      |CREATE TABLE `one` (
```
[the remark this answers is not in the document]
Comment (arkadius) [reply engine/flink/components/base-tests/src/test/scala/pl/touk/nussknacker/engine/flink/table/sink/TableFileSinkTest.scala]: change to more meaningful names in this case and others

File: engine/flink/components/base-tests/src/test/scala/pl/touk/nussknacker/engine/flink/table/sink/TableFileSinkTest.scala [hunk 3]
```diff
@@ -21,13 +22,14 @@ import java.nio.charset.StandardCharsets
...
 
-class TableFileSinkTest extends AnyFunSuite with FlinkSpec with Matchers with PatientScalaFutures {
+class TableFileSinkTest extends AnyFunSuite with FlinkSpec with Matchers with PatientScalaFutures with LoneElement {
 
   import pl.touk.nussknacker.engine.flink.util.test.FlinkTestScenarioRunner._
   import pl.touk.nussknacker.engine.spel.SpelExtension._
 
   private lazy val outputDirectory1 = Files.createTempDirectory(s"nusssknacker-${getClass.getSimpleName}-1")
   private lazy val outputDirectory2 = Files.createTempDirectory(s"nusssknacker-${getClass.getSimpleName}-2")
+  private lazy val outputDirectory3 = Files.createTempDirectory(s"nusssknacker-${getClass.getSimpleName}-3")
```
[the remark this answers is not in the document]
Comment (arkadius) [reply engine/flink/components/base-tests/src/test/scala/pl/touk/nussknacker/engine/flink/table/sink/TableFileSinkTest.scala]: yes, we should

**Review by raphaelsolarski — APPROVED** [review]

**Pull request merged**